src/main/java/com/school/schedule/app/School.java

```diff
@@ -1,17 +1,23 @@
 package com.school.schedule.app;
 
+import java.util.List;
+
 public class School {
     private String name;
     private String address;
     private String city;
     private float telephone;
+    private List students;
```

Comment:
Rather than expecting a list of Students and a list of Teachers in the constructor, I think it may be cleaner to initialize these attributes each to a new List<Student>. Then the constructor will not need to accept either list.

Reply:
fixed it in the new commit!